src/server/page_server.cpp

```diff
@@ -192,7 +195,7 @@ page_server::connection_handler::receive_oldest_active_mvccid (tran_server_conn_
 
   const auto oldest_mvccid = *reinterpret_cast<const MVCCID *const> (a_sp.pull_payload().c_str());
 
-  m_ps.m_pts_mvcc_tracker.update_oldest_active_mvccid (get_channel_id (), oldest_mvccid);
+  m_ps.m_pts_mvcc_tracker.update_oldest_active_mvccid (m_underlying_channel_id, oldest_mvccid);
```

Comment:
Why don't you use the accessor, get_channel_id(), but the variable directly?

Reply:
I will replace it with the accessor.